src/lib/cli/format.js

```diff
@@ -167,3 +167,18 @@ export function getGlyphForStatus( status: string, runningSprite: RunningSprite
 			return chalk.green( '✕' );
 	}
 }
+
+// Format Search and Replace values to output
+export const formatSearchReplaceValues = ( values, message ) => {
+	// Convert single pair S-R values to arrays
+	const searchReplaceValues = typeof values === 'string' ? [ values ] : values;
+
+	const formattedOutput = searchReplaceValues.map( pairs => {
+		const [ from, to ] = pairs.split( ',' );
```

Comment:
What do you think about trimming whitespace proper to output, like so?
```suggestion
		const [ from, to ] = pairs.split( ',' ).map( s => s.trim() );
```

Reply:
Oop, good catch! Added!